### PR TITLE
okazaki p2 pathology detail updates

### PR DESCRIPTION
**Pull request by bullenc**

### CODE REVIEW

File: src/gdcdictionary/schemas/pathology_detail.yaml [hunk 1]
```diff
@@ -1012,6 +1057,60 @@ properties:
...
+      - After Chemotherapy
+      - After Study Enrollment
+      - Childhood
+      - End of Consolidation Therapy
+      - End of Treatment Course
+      - End of Treatment Course 1
+      - End of Treatment Course 2
+      - First Complete Response
+      - First Treatment
+      - Follow-up
```
Comment (wwysoc2) [inline src/gdcdictionary/schemas/pathology_detail.yaml:1088]: Should this be "Follow-Up"?  We can keep it lowercase if the other nodes have it like this.

**Review by wwysoc2 — APPROVED** [review]
Approved, but look at that one "Follow-up" value.  I'll leave the course of action up to you.

File: src/gdcdictionary/schemas/pathology_detail.yaml (hunk 1 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/pathology_detail.yaml:1088]: Yeah the letter case was set on the FollowUp node funny enough, which was then inherited by OtherClinicalAttribute. I think for sake of consistency we should keep as is and for other nodes, and then we can deprecate+migrate in the future.

**Review by ProfOak — APPROVED** [review]

**Review by stilesj-uchicago — APPROVED** [review]
LGTM

**Review by wteouchicago — APPROVED** [review]

**Pull request merged**